server/protocols/webadmin/webadmin-data/pom.xml

```diff
@@ -87,6 +87,12 @@
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
         </dependency>
+        <dependency>
+            <groupId>commons-lang</groupId>
+            <artifactId>commons-lang</artifactId>
```

Comment:
you can use `commons-lang3` which is already declared in the root pom. Or maybe you don't need it at all.